samples/ai-gw-mcp-claude-desktop/configure-claude.sh

```
@@ -0,0 +1,135 @@
+    ;;
+esac
+
+CLAUDE_CONFIG="${CLAUDE_CONFIG_DIR}/claude_desktop_config.json"
+
+if [ -z "$CLAUDE_CONFIG_DIR" ] || [ ! -d "$CLAUDE_CONFIG_DIR" ]; then
+  print_error "Claude Desktop config directory not found at: ${CLAUDE_CONFIG_DIR}"
+  echo ""
+  echo "Is Claude Desktop installed? Download from: https://claude.ai/download"
+  echo ""
+  echo "If installed in a non-standard location, manually add this to claude_desktop_config.json:"
+  print_manual_config() { true; }
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)